test/gurka/test_hard_exclusions_ferry.cc

```diff
@@ -0,0 +1,86 @@
+#include "gurka.h"
```

Comment:
this can probably go into `test_ferry_connections.cc`

Reply:
I added it as a seperate file since others are also seperate for hard exclusions like unpaved etc. What do you think?

Reply:
we have the problem of having too many tests in separate files. it makes the build times very long because each test file is a separate executable that must be built and run. for that reason, where we can, we like to put tests into existing test files, whichever ones that most closely match the area of functionality that we are testing.

Reply:
Moved!